blosc/schunk.c

```diff
@@ -354,9 +356,9 @@ int64_t blosc2_schunk_to_buffer(blosc2_schunk* schunk, uint8_t** dest, bool* nee
 
 /* Write an in-memory frame out to a file. */
 int64_t frame_to_file(blosc2_frame_s* frame, const char* urlpath) {
-  FILE* fp = fopen(urlpath, "wb");
-  size_t nitems = fwrite(frame->cframe, (size_t)frame->len, 1, fp);
-  fclose(fp);
+  FILE* fp = frame->schunk->storage->udio->open(urlpath, "wb", NULL);
```

Comment:
Will open always return `FILE *`? Perhaps it should be `void *fp`. Because if you have a memory stream, it might not return `FILE *` but would instead return a pointer to internal struct about memory allocation etc.

Reply:
Nop, you are right. This plugin is specially designed to manage different types of io formats (s3, db ...). Thanks!